sdk/storage/Azure.Storage.Blobs/samples/Sample02_Auth.cs

```diff
@@ -169,7 +169,7 @@ public async Task SharedAccessSignatureAuthAsync()
 
             // Try to create a new container (which is beyond our
             // delegated permission)
-            StorageRequestFailedException ex = 
+            StorageRequestFailedException ex =
```

Comment:
Would anyone be opposed to adding a check style rule for starting lines with the operator?  I think it's more readable.

```cshare
  StorageRequestFailedException ex
      = Assert.ThrowsAsync<StorageRequestFailedException>(
async () => await service.CreateBlobContainerAsync(Randomize("sample-container")));
```